build/openresty/wasmx/rules.bzl

```diff
@@ -0,0 +1,80 @@
+load("//build/openresty/wasmx:wasmx_repositories.bzl", "wasm_runtimes")
+
+wasmx_configure_options = select({
+    "@kong//build/openresty/wasmx:wasmx_on_linux": [
+        # neded for wasmx module
+        # although this is centos7 specific, the flag will work on any GNU linker
+        # we place it here to skip macos, which uses darwin ld
+        # https://github.com/Kong/ngx_wasm_module/commit/e70a19f53e1dda99d016c5cfa393652720959afd
+        "--with-ld-opt=\"-Wl,--allow-multiple-definition\"",
```

Comment:
The comment says the flag is needed, but I'd like to make sure of it, or else I don't think this is a flag we should be keeping around. Do we need supporting CentOS 7? Because for the module, the only reason we used it was to [bypass a linker issue on CentOS 7](https://github.com/Kong/ngx_wasm_module/blob/main/assets/release/Dockerfiles/Dockerfile.amd64.centos7#L28-L31). Is there another platform needing it, and if so, maybe we should pin the flag to it.

Reply:
We do support rhel7 (which is built in centos7). But I also lean to reduce using it if not necessary, i will change it use a CLI flag to enable it only in rhel7 build time.